tox.ini

```diff
@@ -15,9 +15,9 @@ commands =
 deps =
     -rrequirements/test.txt
 setenv =
+    CUDA_VISIBLE_DEVICES=0
```

Comment:
Without setting `CUDA_VISIBLE_DEVICES=0`, we get an error:
```
tensorflow.python.framework.errors_impl.InvalidArgumentError: Device ordinals must be set for all virtual devices or none. But the device_ordinal is specified for 1 while previous devices didn't have any set.
```
We didn't have to set this with TF 2.9. Is it specific to TF 2.10? Is there a better way to handle this?

Reply:
If would like to run these tests with only 1 GPU, we can also change the runner we use in the workflow config. Currently set to `2GPU`

https://github.com/NVIDIA-Merlin/models/blob/e2276f44e77390b55462e2bd83b6f10f212d9aec/.github/workflows/gpu-ci.yml#L15